configs/sst_cs_software_management-unwanted.yaml

```diff
@@ -0,0 +1,43 @@
+document: feedback-pipeline-workload
```

Comment:
Can you please change this to:

```
document: feedback-pipeline-unwanted
```

The [exclusion list](https://github.com/minimization/content-resolver/blob/master/config_specs/exclusion.yaml) is a different config type.

Reply:
Updated, thanks!